### PR TITLE
docs: add contribution guidelines and pull request template

### PR DESCRIPTION
**Pull request by nightt5879**
## Summary

- Add an English contribution guide covering community language, development checks, pull request expectations, and AI-assisted contributions.
- Add a pull request template for validation evidence, AI disclosure, and contributor responsibility.
- Establish a lightweight and consistent contribution process for the Drawnix community.

## Related Issue

None.

## Validation

- [x] Applicable checks pass: lint, format check, tests, and build
- [x] Relevant E2E and manual tests are complete or not applicable

Evidence, screenshots, or compatibility notes:

- `npm run lint` passed locally.
- `npm run test` passed locally: 4 projects and 29 tests.
- `npm run build` passed locally: 4 projects.
- `npm run format:check` reported existing Windows workspace formatting differences across 128 files; neither added Markdown file was reported. The same check passed in Linux CI.
- The `main` CI job passed, including lint, format check, affected tests/builds, and E2E.
- E2E and manual product testing are not applicable to this documentation-only change.

## AI Assistance

Did you use AI tools to generate or substantially modify code, tests, or documentation in this pull request?

- [ ] No
- [x] Yes

If yes:

- **Tool/model:** Codex / GPT-5.6 Sol
- **AI-assisted work:** I used Codex to review the current repository workflow and settings and to draft and refine `CONTRIBUTING.md` and the pull request template based on my requirements.
- **My review and validation:** I reviewed the complete contents of both documents, confirmed the policy and scope, and approved the final drafts. I also reviewed the repository commands and settings and the reported results from the local lint, tests, and builds.

- [x] I reviewed and understand the AI-assisted changes and can explain and maintain them

## Checklist

- [x] The title and description are in English, and the change is focused
- [x] I reviewed and understand every submitted change
- [x] Tests, documentation, and translations were updated when needed

### CODE REVIEW

**Comment by nightt5879** [comment]
I accidentally clicked “Request Copilot review.” Please ignore it and any follow-up actions or comments it may generate.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+npm run lint
+npm run format:check
+npm run test
+npm run build
+```
+
+For interaction changes, install the Playwright browsers and run the E2E tests:
+
+```bash
+npx playwright install
```
[the remark this answers is not in the document]
Comment (nightt5879) [reply CONTRIBUTING.md:38]: I think this overthinks a rare edge case

**Review by pubuzhixing8 — APPROVED** [review]
Thanks for the contribution! This looks good overall. Approved. As a non-blocking follow-up, please consider adding a link to CONTRIBUTING.md in both README.md and README_en.md so contributors can discover the guide more easily.

**Comment by nightt5879** [comment]
Thanks! I’ve added the `CONTRIBUTING.md` link to both README files. Could you take one more look? If everything looks good, I’ll merge this PR and then set up the same contribution process for Plait.

**Comment by pubuzhixing8** [comment]
LGTM!
